gcc/config/rs6000/t-hipperos

```diff
@@ -0,0 +1,50 @@
+# Multilibs for PowerPC HIPPEROS targets.
+#
+# Copyright (C) 2004-2017 Free Software Foundation, Inc.
+#
+# This file is part of GCC.
+#
+# GCC is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GCC is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GCC; see the file COPYING3.  If not see
+# <http://www.gnu.org/licenses/>.
+
+MULTILIB_OPTIONS =
+MULTILIB_DIRNAMES =
+MULTILIB_MATCHES =
+MULTILIB_EXCEPTIONS =
+MULTILIB_REQUIRED =
+
+MULTILIB_OPTIONS += mcpu=e500mc/mcpu=e5500/mcpu=e6500
+MULTILIB_DIRNAMES += me500mc me5500 me6500
+
+MULTILIB_OPTIONS += m32/m64
```

Comment:
I don't think we need the m32 variant.
We anyway use another compiler for U-Boot for PowerPC.